src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -1,16 +1,25 @@
 from enum import Enum
 from typing import Optional
 
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID, extract_use_case_id
+
 
 class AggregationOption(Enum):
     HIST = "hist"
+    TEN_SECOND = "ten_second"
 
 
 METRIC_ID_AGG_OPTION = {
     "d:transactions/measurements.fcp@millisecond": AggregationOption.HIST,
     "d:transactions/measurements.lcp@millisecond": AggregationOption.HIST,
 }
 
+USE_CASE_AGG_OPTION = {UseCaseID.CUSTOM: AggregationOption.TEN_SECOND}
+
 
 def get_aggregation_option(metricId: str) -> Optional[AggregationOption]:
```

Comment:
Damn I can't believe I put `metricId` instead of `metric_id` here, could you fix it in this pr for me? Sorry

Reply:
Yep!